tests/sentinel/tests/03-runtime-reconf.tcl

```diff
@@ -44,5 +44,17 @@ test "Sentinel Set with other error situations" {
 
    # unknown parameter option
    assert_error "ERR Unknown option or number of arguments for SENTINEL SET 'fakeoption'" {S 0 SENTINEL SET mymaster fakeoption fakevalue}
+
+   # save new config to disk failed
+   set info [S 0 SENTINEL master mymaster]
+   set origin_quorum [dict get $info quorum]
+   set update_quorum [expr $origin_quorum+1]
+   set sentinel_id 0
+   set configfilename [file join "sentinel_$sentinel_id" "sentinel.conf"]
+   exec chmod 000 $configfilename
+
+   assert_error "ERR Failed to save Sentinel new configuration on disk" {S 0 SENTINEL SET mymaster quorum $update_quorum}
+
+   exec chmod 644 $configfilename
```

Comment:
Nothing serious. If test fails, there is leftover of locked file on filesystem. Maybe: 
res={...}; chmod 644 ; assert_error res

Reply:
As we discussed,  I update the test codes, please take a look. Thanks for your great help on this.